libbeat/scripts/Makefile

```diff
@@ -240,6 +240,7 @@ update: python-env
 	cp ${ES_BEATS}/libbeat/docs/version.asciidoc docs/version.asciidoc
 
 	# Generate index-pattern
+	mkdir -p $(PWD)/etc/kibana/index-pattern
```

Comment:
It would be nice to first remove all index-pattern and then only generate the new ones. This makes sure not old left overs stay in and the generation works as expected. We do the same for example for the config files.